apps/mull-api/src/environments/environment.prod.ts

```diff
@@ -4,9 +4,9 @@ export const environment = {
   production: true,
 
   db: {
-    host: process.env.DB_HOST,
-    username: process.env.DB_USERNAME,
-    password: process.env.DB_PASSWORD,
+    host: process.env.TYPEORM_HOST,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
```

Comment:
Is there a reason why these were renamed? Its the db host and credentials, not the host and credentials for typeorm.

Reply:
Without this, the typeorm seeding won't be able to connect the database, it was taken from [their docs](https://github.com/typeorm/typeorm/blob/master/docs/using-ormconfig.md)

Reply:
I see, that's fine I guess